From 19188367bee87b0084ab34869455155da5cb1731 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 13 Apr 2022 08:47:39 -0700
Subject: [PATCH] meson: Check for libunwind instead of libunwind-generic

This helps it to use llvm unwinder since libunwind-generic is specific
to nongnu libunwind.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 3d3d8b5..4a1963d 100644
--- a/meson.build
+++ b/meson.build
@@ -96,7 +96,7 @@ config_h.set10('HAVE_POLKIT', polkit_dep.found())
 if get_option('libunwind')
   # Force libunwind usage if it's specified to avoid back compiles
   # and backtrace() showing up in builds
-  libunwind_dep = dependency('libunwind-generic', required: true)
+  libunwind_dep = dependency('libunwind', required: true)
   config_h.set('ENABLE_LIBUNWIND', libunwind_dep.found())
   config_h.set('HAVE_UNW_SET_CACHE_SIZE', libunwind_dep.found() and cc.has_header_symbol('libunwind.h', 'unw_set_cache_size', dependencies: [libunwind_dep]))
 endif
-- 
2.35.2

